### DISCUSSION TITLE
可以出一个安装教程吗

### DISCUSSION THREAD

**Discussion by XuanMouren2004** [Johnserf-Seed/TikTokDownload#744] [General]
自从更新了f2之后就开始不会用了，下载也无法正常下载，作者可以抽空出一个视频的安装流程吗，哭了/(ㄒoㄒ)/~~

**Comment by JoeanAmier** [comment]
看文档和 `--help` 基本就会用了。

**Reply by XuanMouren2004** [reply]
主要是我安装完f2之后，跟着文档走，就开始报错了，然后跟着网上的操作，也是不对。
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/143273609/ffde694b-6a7f-44ca-89f8-c673532bef8f)
提示这个报错，我把权限打开了，也是报错这种。
f2 -h

**Reply by Johnserf-Seed** [reply]
> 主要是我安装完f2之后，跟着文档走，就开始报错了，然后跟着网上的操作，也是不对。 ![image](https://private-user-images.githubusercontent.com/143273609/346520876-ffde694b-6a7f-44ca-89f8-c673532bef8f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzY4NDQsIm5iZiI6MTcyMDQzNjU0NCwicGF0aCI6Ii8xNDMyNzM2MDkvMzQ2NTIwODc2LWZmZGU2OTRiLTZhN2YtNDRjYS04OWY4LWM2NzM1MzJiZWY4Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwOFQxMTAyMjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZmEwZmY4NmI4NTUwMDliNmZhNDQ4NWU0ZGJiMmRlYTBmNTJlZjMwMjViMmI2NThlYWVlMjdiYTM2MjJkZTE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.AlrTXnNB8eTuy8uuib_U2edHo4QupgYecTNni9OEa8w) 提示这个报错，我把权限打开了，也是报错这种。 f2 -h

这种情况要么是你python安装的时候没有勾选添加到环境变量，看网上的教程设置一下或者重新安装

**Reply by XuanMouren2004** [reply]
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/143273609/b9adeb3b-dfe8-4bf1-b989-141a04db2495)
我python的环境变量是有的，一切正常，就是无法使用，/(ㄒoㄒ)/~~

**Reply by Johnserf-Seed** [reply]
不要用ps用cmd可以运行吗，那你重新安装一遍f2再看看

**Reply by XuanMouren2004** [reply]
好的，我现在尝试一下。

**Reply by XuanMouren2004** [reply]
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/143273609/68a0262d-14fa-4b22-b100-8960e306c9e5)
重新安装了，现在还是不能运行，😵

**Reply by Johnserf-Seed** [reply]
> ![image](https://private-user-images.githubusercontent.com/143273609/346537803-68a0262d-14fa-4b22-b100-8960e306c9e5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0Mzc2ODksIm5iZiI6MTcyMDQzNzM4OSwicGF0aCI6Ii8xNDMyNzM2MDkvMzQ2NTM3ODAzLTY4YTAyNjJkLTE0ZmEtNGIyMi1iMTAwLTg5NjBlMzA2YzllNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwOFQxMTE2MjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NGRlOTM3NmJkY2Y2MDkzMTA4ZTU4ODYyY2IwNWViNGQ2NDQzNDI4ODJkYmY3ODA4YzZjMWIxYTM1ZDIzMWE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.n6Ic5Bcb6Nfoid5BegV5-ppywjm7WWwc41iKrrzlbHM) 重新安装了，现在还是不能运行，😵

关闭这个终端再运行，如果还是不行就是你系统的问题了

**Reply by XuanMouren2004** [reply]
还是不行的，我系统是Windows11x64专业版，而且还是新安装的系统😢
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/143273609/56ec92e3-e0f4-4e07-b45d-15a5a2003508)

**Reply by Johnserf-Seed** [reply]
那你可以尝试用 `python -m f2 -h`来运行试试

**Reply by XuanMouren2004** [reply]
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/143273609/425a7604-82ce-4aaf-94d0-62488e2d878d)
这个是可以的！但是有具体代码教程吗🫡，我还没这样用过

**Reply by XuanMouren2004** [reply]
直接在教程代码的前面加上 `python -m f2 xx` 就可以的吧

**Reply by Johnserf-Seed** [reply]
> ![image](https://private-user-images.githubusercontent.com/143273609/346539623-425a7604-82ce-4aaf-94d0-62488e2d878d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzgxMDgsIm5iZiI6MTcyMDQzNzgwOCwicGF0aCI6Ii8xNDMyNzM2MDkvMzQ2NTM5NjIzLTQyNWE3NjA0LTgyY2UtNGFhZi05NGQwLTYyNDg4ZTJkODc4ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwOFQxMTIzMjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MzllNzdmOGNhNjg5NWM4MTAxMTRiMDYyNDlkMWZlZmI3NTA2NzMwZDgzMmQ3MWVjYjZmMjE5NTIwMWQ3YTBlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.M6SOX72QRAtEuTTX6BTLPjEbOhaJqYSzn3xQ_0AXV4E) 这个是可以的！但是有具体代码教程吗🫡，我还没这样用过

这个的情况是你的site-packages没有添加到环境变量，所以还是环境变量的问题，你需要添加这个路径
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/40727745/1719ca30-0959-429a-8cc0-71310a0d0ae6)
然后再重启终端应该就可以使用f2的命令了

**Reply by XuanMouren2004** [reply]
麻烦问一下作者用的是什么版本的python我感觉我应该换个版本，我现在用的3.12.4，尝试了一下还是不行的，有可能版本过于高了吧？

**Reply by Johnserf-Seed** [reply]
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/40727745/ab777218-fee0-481a-a2eb-d1e2b990ffed)

**Reply by XuanMouren2004** [reply]
好的，麻烦哥了，辛苦😄

**Reply by XuanMouren2004** [reply]
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/143273609/67c12e6d-f059-436f-8d04-327c2748fbf6)
现在是这样的状态😢，下载不了，然后用之前的方式下载也是只能下载图集，不能下载视频

**Reply by Johnserf-Seed** [reply]
https://johnserf-seed.github.io/f2/question-answer/qa.html#warning-%E6%B2%A1%E6%9C%89%E6%89%BE%E5%88%B0%E7%AC%A6%E5%90%88%E6%9D%A1%E4%BB%B6%E7%9A%84%E4%BD%9C%E5%93%81

**Reply by Johnserf-Seed** [reply]
先把文档看了再运行

**Reply by XuanMouren2004** [reply]
好的，麻烦您了

**Reply by XuanMouren2004** [reply]
```
douyin:

  # 用户主页(非视频链接)
  url: 

  #下载模式(post|like|listcollection|wix合集暂未更新)
  mode: post

  # 视频原声保存(yes|no)
  music: yes

  # 视频封面保存(yes|no)
  cover: yes

  # 视频文案保存(yes|no)
  desc: yes

  #发布时间(all)
  interval: all

  # 作品保存到单独的文件夹(yes|no)
  # 如果设置了保存原声、封面、文案的建议开启该选项
  folderize: yes
```
刚弄了一下配置文件，感觉这个样子就适合我这种小白配置了🫡，和之前的版本比较熟悉
